proof/invariant-abstract/ARM_HYP/Machine_AI.thy

```diff
@@ -274,7 +274,7 @@ lemma no_fail_cleanCacheRange_RAM[simp, wp]:
 
 lemma no_fail_invalidateCacheRange_I[simp, wp]:
   "no_fail \<top> (invalidateCacheRange_I s e p)"
-  by (simp add: invalidateCacheRange_I_def, wp no_fail_invalidateByVA_I)
+  by (simp add: invalidateCacheRange_I_def, wp no_fail_invalidate_I_PoU)
```

Comment:
Nothing to do with this PR, but we're pretty inconsistent with marking the lemmas in this file with `[simp]` and `[wp]`. If that was more consistent then this would have been an even simpler change.

Reply:
True, that would've made it even smaller.